src/Bundler/Processor/ModuleProcessor.php

```diff
@@ -26,7 +26,9 @@ public function peek(string $cwd, ContentState $state): void
 
     public function transpile(string $cwd, ContentItem $item): void
     {
-        $js  = "register('" . $item->module_name . "', function (define, require, module, exports) {\n";
+        $js  = "register("
+            . json_encode($item->module_name, JSON_UNESCAPED_SLASHES)
```

Comment:
This is a fix for probably a bug that exists for which no unit-test has been added.

Reply:
this is just a common XSS fix